guides/v2.3/extension-dev-guide/routing.md

```diff
@@ -1 +1,187 @@
-../../v2.2/extension-dev-guide/routing.md
\ No newline at end of file
+---
+group: extension-dev-guide
+title: Routing
+version: 2.1
```

Comment:
Should be version: 2.3

Reply:
As of yesterday, we no longer require `version`.

Reply:
@keharper,  we still need to merge that PR before this will be true on `master`: https://github.com/magento/devdocs/pull/2750

Reply:
It seems that it was already merged. Good to know this new PR for next Reviews. 

Thanks!